Packs/CommonScripts/Scripts/GetTime/GetTime.yml

```diff
@@ -14,9 +14,13 @@ args:
   default: true
   description: The date to use in the object
 - name: dateFormat
-  description: The date format for the result, can be ISO/GMT/UTC/Locale/Date/Year/Month/Day/DayInWeek/Hours.
+  description: |-
+    The date format for the result, can be ISO/GMT/UTC/Locale/Date/Year/Month/Day/DayInWeek/Hours.
     Note that the response will start from 0, E.G. if the current month is October
     (the 10th month) the result will be 9.
+    Note:
+    In XSOAR versions 6.11.0 or later, or 8.1.0 or later, the time format returned when UTC or GMT is given has changed.
+    Instead of 'Wed Jan 18 2023 12:59:12 GMT+0000 (UTC)', we now get 'Wed, 18 Jan 2023 15:04:41 GMT'.
```

Comment:
```suggestion
    Instead of 'Wed Jan 18 2023 12:59:12 GMT+0000 (UTC)', The output will be 'Wed, 18 Jan 2023 15:04:41 GMT'.
```
```suggestion
    Instead of 'Wed Jan 18 2023 12:59:12 GMT+0000 (UTC)', we now get 'Wed, 18 Jan 2023 15:04:41 GMT'.
```